filters/exclusions.txt

```diff
@@ -264,5 +258,30 @@ _campaign=$popup
 /^##\.ad$/
 !###############
 !
+!
+!### Invalid rules ###
+! This section is for known old invalid rules
+!
+! void.gr
+||www.dwrean.net##a[href="http://www.kratisinow.gr"]
+||www.dwrean.net##iframe[src="http://kratisinow.digitup.eu/widget/widget-artists"]
+!
+! Xfiles
+||the-japan-news.com/modules/js/lib/fgp/fingerprint2.js$script,redirect=fingerprint2.js,important
```

Comment:
If I'm not wrong, this rule should be converted into `redirect=fingerprintjs` - https://github.com/AdguardTeam/Scriptlets/issues/187
So, maybe it shouldn't be excluded?

Reply:
ok